pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[build-system]
+requires = ["setuptools >= 40.8.0", "wheel"]
```

Comment:
@radarhere @hugovk `wheel` shouldn't actually be depended on: https://github.com/pypa/setuptools/pull/3056.

Reply:
This has since been removed in #7248.

Reply:
Ah, I should've checked. Thanks!